src/gui/shell.cpp

```diff
@@ -87,6 +87,8 @@ QString Shell::fontDesc()
 		fdesc += ":b";
 	} else if (font().weight() == QFont::Light) {
 		fdesc += ":l";
+	} else if (font().weight() == QFont::DemiBold) {
+		fdesc += ":sb";
```

Comment:
This check cannot be reached: `font().weight() == QFont::DemiBold`

The `::bold()` function above returns true for semibold fonts too. The ordering needs to be changed, or the conditionals made more specific.

See: https://doc.qt.io/qt-5/qfont.html#bold